.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,33 @@
+name: Feature request
+description: Let us know about something you'd like to see as part of the CLI or Oxide Rust SDK
+labels: ["feature"]
```

Comment:
I think we need to add this feature. could we automatically label issues according to CLI / SDK as well?

Reply:
I'll add the label! Looks like automatic labels are all or nothing though :( https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax